oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreService.java

```diff
@@ -276,6 +290,8 @@ protected void activate(ComponentContext context, Configuration config) throws E
         cancelInvalidationFeature = Feature.newFeature(FT_NAME_CANCEL_INVALIDATION, whiteboard);
         docStoreFullGCFeature = Feature.newFeature(FT_NAME_FULL_GC, whiteboard);
         docStoreEmbeddedVerificationFeature = Feature.newFeature(FT_NAME_EMBEDDED_VERIFICATION, whiteboard);
+        docStoreFullGCModeGapOrphansFeature = Feature.newFeature(FT_NAME_FULL_GC_MODE_GAP_ORPHANS, whiteboard);
+        docStoreFullGCModeEmptyPropertiesFeature = Feature.newFeature(FT_NAME_FULL_GC_MODE_EMPTY_PROPERTIES, whiteboard);
```

Comment:
(same as above comment - I'll not repeat this comment further though)

Reply:
fixed, thank you for the comment.